concepts/src/services/logging.ts

```diff
@@ -1,39 +1,22 @@
-import {
-  createLogger,
-  transports,
-  format,
-  config,
-  LeveledLogMethod,
-} from "winston";
+import { createLogger, transports, format } from "winston";
 
 // This roughly matches the logback config used by the Scala apps
 const formatter = format.combine(
   format.timestamp({ format: "HH:mm:ss.SSS" }),
   format.errors({ stack: true }),
-  format.printf(
-    ({ level, message, timestamp }) =>
-      `${timestamp} ${level.toUpperCase()} - ${message}`
-  )
+  format.printf(({ level, message, timestamp, stack }) => {
+    let log = `${timestamp} ${level.toUpperCase()} - ${message}`;
+    if (stack) {
+      log += "\n" + stack;
```

Comment:
Given we have more fine-grained control over the logging here, is there anything we can do to make these a bit nicer in the logging cluster? e.g. using `r` instead of `n`.

Reply:
Ah yes, good idea - ideal world I would like us to switch to proper structured logging (probably https://www.elastic.co/guide/en/ecs/current/ecs-reference.html), but not right now